src/backend/distributed/shared_library_init.c

```diff
@@ -83,7 +83,9 @@ _PG_init(void)
 	if (!process_shared_preload_libraries_in_progress)
 	{
 		ereport(ERROR, (errmsg("Citus can only be loaded via shared_preload_libraries"),
-						errhint("Add citus to shared_preload_libraries.")));
+						errhint("Add citus to shared_preload_libraries in postgresql.conf"
+								"in master and worker nodes. Note that citus should be at"
```

Comment:
we are trying to avoid using **node** to mention master and workers. You could just say master and workers.